data/themes/darktable.css

```diff
@@ -521,7 +521,7 @@ overshoot.right
 /* Adjust add button in picker module */
 .picker-module button
 {
-  padding: 0 3px;
+  padding: 0 0px;
```

Comment:
It's worse when panel is not reduce too much. And having 3px as actually in master works with your PR AND when width is ok to see all information correctly.

Reply:
I didn't remove the left-right padding on the button to save space, but because it changes the shape from square to rectangular, which I felt didn't look as nice. I'd prefer to add the padding to the label with the numbers instead, if that is alright with you?

When the panel is wide, the padding has little effect because the label is centered in the available space, so there's additional empty space to the left and right anyway. So this really only impacts narrow panels, but if you think it's better, I don't mind.

Reply:
On label it's indeed better. I didn't thought about button shape. Good catch.

Reply:
Done! Thank you.